skiko/src/awtMain/kotlin/org/jetbrains/skiko/windows/JumpList.kt

```diff
@@ -0,0 +1,156 @@
+package org.jetbrains.skiko.windows
+
+import kotlinx.coroutines.suspendCancellableCoroutine
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+import org.jetbrains.skiko.hostOs
+
+/**
+ * Provides access to Windows Jump List features
+ */
+object JumpList {
```

Comment:
@igordmn I've noticed that everywhere in Skiko where a static class is defined a companion object is used. Should I also do it here?

```kotlin
class JumpList private constructor() {
    companion object {
        fun isSupported() ...
    }
}
```

Usage-wise seems pretty much the same to me.

Reply:
I think it could _theoretically_ work window-wise **if** the window had a separate AppUserModelID, **but** I believe this is discouraged. We also don't set any AppUserModelID at the moment, so the system assigns one automatically to the process itself.